sass/components/_emojisprite.scss

```diff
@@ -3343,4 +3343,5 @@
 .emoji-270d-1f3fc { background-position: -3696px -660px; }
 .emoji-270d-1f3fd { background-position: -3696px -726px; }
 .emoji-270d-1f3fe { background-position: -3696px -792px; }
-.emoji-270d-1f3ff { background-position: -3696px -858px; };
+.emoji-270d-1f3ff { background-position: -3696px -858px; }
+.emoji-mattermost { background-position: -NaNpx -NaNpx; };
```

Comment:
I'm not sure how to fix this easily in emoji_gen.mjs since the code doesn't know what's a custom emoji by the time it reaches the CSS generation, but I don't think this should have any impact in practice.

Reply:
maybe skip it from adding if it's category is custom?

I can't add it as a comment/suggestion in the file (don't know why github does't allow me)
but in emoji_gen.js, line 181
```js
if (emoji.category !== 'custom') {
  emojiFilePositions.set(file, `-${emoji.sheet_x * EMOJI_SIZE_PADDED}px -${emoji.sheet_y * EMOJI_SIZE_PADDED}px;`);
}
```

Reply:
Oh yeah. Of course we can just stop having it added to that array to begin with.